modules/shell/command.go

```diff
@@ -84,32 +91,24 @@ func RunCommandAndGetOutputE(t *testing.T, command Command) (string, error) {
 }
 
 // This function captures stdout and stderr while still printing it to the stdout and stderr of this Go program
-func readStdoutAndStderr(t *testing.T, stdout io.ReadCloser, stderr io.ReadCloser) (string, error) {
+func readStdoutAndStderr(t *testing.T, outputs ...io.ReadCloser) (string, error) {
 	allOutput := []string{}
 
-	stdoutScanner := bufio.NewScanner(stdout)
-	stderrScanner := bufio.NewScanner(stderr)
+	scanners := []bufio.Scanner{}
+	for i := range outputs {
+		scanners = append(scanners, *bufio.NewScanner(outputs[i]))
+	}
 
-	for {
-		if stdoutScanner.Scan() {
-			text := stdoutScanner.Text()
+	for i := range scanners {
```

Comment:
Doesn't this just loop over each scanner once and exit? The original `for` loop kept going infinitely until none of the scanners had any more data or it hit an error.